src/gbd_mapping/risk_factor_template.py

```diff
@@ -72,7 +72,7 @@ class RiskFactors(GbdRecord):
                  'diet_low_in_omega_6_polyunsaturated_fatty_acids', 'diet_high_in_trans_fatty_acids',
                  'diet_high_in_sodium', 'low_physical_activity', 'occupational_risks', 'occupational_carcinogens',
                  'occupational_asthmagens', 'occupational_particulate_matter_gases_and_fumes', 'occupational_noise',
-                 'occupational_injuries', 'occupational_ergonomic_factors', 'childhood_sexual_abuse',
+                 'occupational_injuries', 'occupational_ergonomic_factors', 'sexual_violence_against_children',
```

Comment:
I kinda thought part of this gbd update was to strip out stuff that we don't actually expect to use.